setup.py

```diff
@@ -9,6 +9,7 @@
 import urllib2
 
 from setuptools import setup, find_packages, Extension
+from wheel import egg2wheel
```

Comment:
Can we import this where it is used? Otherwise if the user doesn't have wheel installed and just wants to install NuPIC then we will get an error.